source/slang/hlsl.meta.slang

```diff
@@ -2320,10 +2318,9 @@ vector<T,N> GetAttributeAtVertex(vector<T,N> attribute, uint vertexIndex);
 __generic<T : __BuiltinType, let N : int, let M : int>
 [__readNone]
 __target_intrinsic(hlsl)
-__target_intrinsic(glsl, "$0[$1]")
+__target_intrinsic(GL_NV_fragment_shader_barycentric, "$0[$1]")
+__target_intrinsic(GL_EXT_fragment_shader_barycentric, "$0[$1]")
```

Comment:
Is this the correct way to conditionally require a GL extension for a HLSL function?

I was not sure, but it seems to work.

Reply:
Yes, this look correct to me.